tests/files/read_xyc_test.xyc

```diff
@@ -0,0 +1,4 @@
+1 2 3
```

Comment:
Maybe we can look into using io for tests using small files.
I looked it up and it seems you can "Mock"  files.
https://docs.python.org/3/library/io.html

Reply:
example: 

```
    def test_read_waqua_xyz_Marlon_io_test(self):
        """
        Read WAQUA xyz file columns 1 and 2.
        """
        data_string = '''\
                        "A", "B", "C"
                        1, 2, 3
                        4, 5, 6
                        7, 8, 9
                        10, 11, 12
                        '''
        string_io = io.StringIO(data_string)
        string_io.name = "read_waqua_xyz_test.xyc"
        col = (1,2)
        data = dfastmi.io.read_waqua_xyz(string_io, col)
        datar = numpy.array([[ 2., 3.], [ 5., 6.], [ 8., 9.], [11., 12.]])
        print("data reference: ", datar)
        print("data read     : ", data)
        assert numpy.shape(data) == (4,2)
        assert (data == datar).all() == True
```

Reply:
will be tackled in https://issuetracker.deltares.nl/browse/DFAST-86